instrumentation/opentelemetry_phoenix/lib/opentelemetry_phoenix.ex

```diff
@@ -6,8 +6,9 @@ defmodule OpentelemetryPhoenix do
                       doc: "The endpoint prefix in your endpoint."
                     ],
                     adapter: [
-                      type: {:in, [:cowboy2, :bandit, nil]},
-                      default: nil,
+                      type: {:in, [:cowboy2, :bandit]},
+                      default: :cowboy2,
+                      required: true,
```

Comment:
Yeah it's too bad to have a breaking change here, but I agree it makes more sense to force the choice instead of having to also have a default thing built in here.

Reply:
Right. With bandit support now it just doesn't make sense to not force it and this is a breaking release anyhow, so time to do it.